### PR TITLE
Vagrant First Steps

### PR DESCRIPTION
**Pull request by FGS-BMarkham**

### CODE REVIEW

**Comment by FGS-CThomas** [comment]
I think this is easy to follow and well done.  It's not trying to be a tutorial but I feel like I could still follow the steps myself if so inclined.  Consider holding to 1st person plural (as in the first part) or singular (as in the latter bits) for consistency's sake, though I recognize that there's a departure from group work in actuality. 

Further, consider a bit more in the way of explanation of why the constraints that prompted this existed in the first place (something like "we'd like to use this to automate environment spin up in our testing process")

**Comment by FGS-BMarkham** [comment]
Please review latest version.

NOTE: I need to squash this before it gets merged.

**Comment by FGS-BMarkham** [comment]
Please review latest version. Addresses existing criticism and updated to reflect some Packer research.

NOTE: I need to squash this before it gets merged.

**Comment by FGS-BMarkham** [comment]
Also, title & prettification suggestions are welcome.

File: _posts/2016-01-18-Vagrant-First-Steps.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+### Here Are The Steps We Followed
+
+1. We needed a Windows Server ISO, so we fired up our MSDN subscription and downloaded `en_windows_server_2016_technical_preview_4_x64_dvd_7258292.iso` because we were feeling adventurous.
+
+2. We used Hyper-V Manager to create a `Generation 2` Virtual Machine named `WinServ2016CTP4Core`, with `2048` MB of (dynamic) startup memory, attached to an `External` virtual switch, with a new VHDX with the same name as the VM.
+
+3. We mounted the ISO to the VM, fired it up, and installed Windows Server 2016 Technical Preview 4 in `Core` mode.
+
+4. we gave the `Administrator` account a random (complex) password. (You can change it later with `net user`, as [this post](http://www.thewindowsclub.com/net-user-command-windows) shows.)
```
Comment (FGS-CThomas) [inline _posts/2016-01-18-Vagrant-First-Steps.md]: Change `we` to `We` for caps agreement w/ the rest of the  list

**Comment by FGS-CThomas** [comment]
Other than that comment about Step 4 -- :shipit:

**Pull request merged**